templates/zerver/api/webhook-walkthrough.md

```diff
@@ -204,7 +205,7 @@ After which you should see:
 
 Using either method will create a message in Zulip:
 
-![Image of Hello World webhook message](images/helloworld-webhook.png)
+<img class="screenshot" src="/static/images/api/helloworld-webhook.png" />
```

Comment:
@eeshangarg any idea why the Markdown image syntax doesn't work in the /api system?